opengrok-indexer/src/main/java/org/opengrok/indexer/configuration/Configuration.java

```diff
@@ -1336,6 +1339,14 @@ public void setDisabledRepositories(Set<String> disabledRepositories) {
         this.disabledRepositories = disabledRepositories;
     }
 
+    public Set<String> getAuthenticationTokens() {
```

Comment:
Should not we return a copy or at least return an unmodifiable collection? Theoretically, someone could change the content easily.

Reply:
Good catch ! Definitely something to worry about given these are access tokens.

Reply:
Similar to https://github.com/oracle/opengrok/pull/3312#discussion_r511657948 but perhaps better to solve it here also